packages/api-reference/src/components/Layouts/ModernLayout.vue

```diff
@@ -65,7 +66,10 @@ watch(hash, (newHash, oldHash) => {
     </template>
     <template #sidebar-end>
       <div class="darklight-reference">
-        <OpenApiClientButton />
+        <OpenApiClientButton
+          :integration="configuration._integration ?? 'vue'"
```

Comment:
Users can pass `null` to avoid having any integration identifier passed down, so we need another check here I think. Maybe this?

```suggestion
          :integration="configuration._integration === undefined ? 'vue' : configuration._integration"
```

Reply:
going to move it inside the component instead